config-tree.js

```diff
@@ -0,0 +1,9 @@
+const fs = require("fs");
+const dirTree = require("directory-tree");
+
+const tree = dirTree('./config/v3', {extensions:/\.json$/});
+
+fs.writeFile('./config/config-tree.json', JSON.stringify(tree, null, 2), function (err) {
+    if (err) throw err;
+    console.log('Saved!');
+});
```

Comment:
this is a good beginning for a script, looks good. Please take note that it would generate the file so consider running it as a step, that result (the file) will be later used to commit automatically on `config` folder. Take a look at  reference: https://github.com/binary-com/deriv-custom-terminology/blob/main/.github/workflows/custom_terminology_to_AWS.yml#L9-L22